tutorials/2_sprite_animations_gestures/README.md

```diff
@@ -0,0 +1,241 @@
+
+```dart
+  // One sprite for each button state
+  late Sprite pressedButton;
+  late Sprite unpressedButton;
+  // Just like our robot needs its position and size, here we create two
+  // variables for the button as well
+  final buttonPosition = Vector2(200, 120);
+  final buttonSize = Vector2(120, 30);
+  // Simple boolean variable to tell if the button is pressed or not
+  bool isPressed = false;
+```
+
+Next we load our two sprites:
+
+```dart
+  @override
+  Future<void> onLoad() async {
+    // runningRobot loading omited
+
+    // Just like we have a `loadSpriteAnimation` function, here we can use
+    // the `loadSprite`. To use it we just need to inform the assets path
+    // and the position and size which will define the section of the image
+    // that we want. If we wanted to have a sprite with the full image, `srcPosition`
+    // and `srcSize` could just be omited
+    unpressedButton = await loadSprite(
+      'buttons.png',
+      // `srcPosition` and `srcSize` here tells `loadSprite` that we want
+      // just a rect (starting at (0, 0) with the dimensions (60, 20)) of the image
+      // which gives us only the first button
+      srcPosition: Vector2.zero(),
+      srcSize: Vector2(60, 20),
+    );
+
+    pressedButton = await loadSprite(
+      'buttons.png',
+      // Same thing here, but now a rect starting at (0, 20)
+      // which gives us only the second button
+      srcPosition: Vector2(0, 20),
+      srcSize: Vector2(60, 20),
+    );
+  }
+```
+
+Finally, we just render it on the game `render` function:
+
+```dart
+  @override
+  void render(Canvas canvas) {
+    // Running robot render omited
+
+    if (isPressed) {
+      pressedButton.render(canvas, position: buttonPosition, size: buttonSize);
+    } else {
+      unpressedButton.render(canvas,
+          position: buttonPosition, size: buttonSize);
+    }
+  }
+```
+
+You now should see the button on the screen, but right now, it is pretty much useless as it has no action at all.
+
+So, to change that, we will now add some interactivity to our game and make the button tapable/clickable.
+
+Flame provides several input handlers, which you can check with more in depth on the [input documentation of Flame](https://github.com/flame-engine/flame/blob/main/doc/input.md). For this example we will be using the `TapDetector` which enables us to detect taps on the screen, as well as mouse click when running on web or desktop.
+
+All of Flame input detectors are mixins which can be added to your game, which when applied, enables you to override listener methods related to that detector. In our example, we will need to override three methods:
+
+ - `onTapDown`: Called when touch/click has started. i.e. the user just touced the screen or clicked the mouse button.
+ - `onTapUp`: Called when the touch/click has stop ocurring because the event was release. i.e. the user lifted the finger from the screen or released the mouse button.
```

Comment:
```suggestion
 - `onTapUp`: Called when the touch/click has stop occurring because the event was released, i.e., the user lifted the finger from the screen or released the mouse button.
```